altislife.sql

```diff
@@ -233,15 +241,8 @@ CREATE TABLE IF NOT EXISTS `wanted` (
       ON UPDATE CASCADE ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
--- --------------------------------------------------------
---
--- Creates default user `arma3` with password `changeme` unless it already exists
--- Granting permissions to user `arma3`, created below
--- Reloads the privileges from the grant tables in the MySQL system database.
---
 
-CREATE USER IF NOT EXISTS `arma3`@`localhost` IDENTIFIED BY 'changeme';
-GRANT SELECT, UPDATE, INSERT, EXECUTE ON `altislife`.* TO 'arma3'@'localhost';
+GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER, CREATE VIEW, SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, EXECUTE ON `altislife`.* TO 'arma3'@'localhost';
```

Comment:
Please revert these changes. It is intended for the `arma` user to not have DELETE permissions

Reply:
in addition to other reply: WHY?

Reply:
For security purposes